src/Mado/QueryBundle/Objects/FilteringObject.php

```diff
@@ -59,4 +59,20 @@ public function getOperatorSign()
     {
         return $this->getOperator()['meta'];
     }
+
+    public function isListOperator()
```

Comment:
Can you please add the Return type?

Reply:
Thank you!!! I always forget return type